pep-9999.rst

```diff
@@ -0,0 +1,217 @@
+
+    [authentication]
+
+    [authentication.pypi]
+    username = "__token__"
+    password = "pypi-AgEIcHlwaS5vcmcCJDJjMmI5NDJmLWIzNzQtNDc3ZC04ZmRkLTM4NDEzY2M1ZTU5MAACJXsicGVybWlzc2lvbnMiOiAidXNlciIsICJ2ZXJzaW9uIjogMX0AAAYgV91r8iEUUz7Q5xbsfi_KGiO3wrJR2_IOOpnFyLo62bk"
+
+    [authentication.myrepo1]
+    username = "foo"
+    password = "YmFyCg=="
+
+
+Instead of specifying the credentials in this file, Keyring can be used::
+
+    keyring set https://pypi.org/simple/ __token__
+    keyring set <repository URL> <private-repository username>
+
+
+A repository's credentials has the following attributes::
+
+    ==================  ===========  ===============================================
+    Attribute           Type         Description
+    ==================  ===========  ===============================================
+    cert                str          Path to certificate authority bundle (*.pem)
+    cert-client         str          Path to client certificate (*.pem)
+    password            str          The password for the repository, base64-encoded. When token is used, enter it here in clear-text.
+    proxy_password      str          The password for the proxy server, base64-encoded.
+    proxy_username      str          The username for the proxy server
+    username            str          The username for the repository. When token is used, set this to ``__token__``.
+    ==================  ===========  ===============================================
+
+
+Package discovery
+-----------------
+
+The PyPI repository (``repositories.pypi``) does not have to be specified in the ``config.toml``. If not specified, it will be used as the first package index by default. However, by explicitly defining it, the repository attributes can be used and it can be ordered differently.
+
+The order of the repository definitions in ``config.toml`` shall be respected when searching for packages. This allows custom package indexes to take precendence over e.g. a default PyPI repository and the use of package shadowing.
+
+
+Configuration file discovery
+----------------------------
+
+The locations of the configuration files vary slightly across platforms. You may have per-user, per-virtualenv or global (shared amongst all users) configuration:
+
+If multiple configuration files are found, then they are combined (merge and override) in the following order:
+
+1. The global file is read
+2. The per-user file is read
+3. The virtualenv-specific file is read
+
+Each file read overrides any values read from previous files.
+
+
+Global
+''''''
+
+* Unix: ``$HOME/.config/python/``
+* macOS: ``$HOME/.config/python/``
+* Windows: ``%APPDATA%\python\``
+
+
+Per-user
+''''''''
+
+* Unix: ``$HOME/.config/python/``
+* macOS: ``$HOME/.config/python/``
+* Windows: ``%APPDATA%\python\``
```

Comment:
```suggestion
* Windows: ``%APPDATA%\Python\`` (note, this is Roaming app data, not Local)
```

Reply:
Roaming meaning it's for all users, right?

Would it be better to store this in `$HOME/.config/python` even on Windows?
I've seen some applications use the "dot folders" even in Windows. Or this this going too far outside of Windows conventions?

Reply:
Roaming means that if the user logs into another machine (under some circumstances), the file will automatically roam to the new machine. Putting it into LocalAppData or just under UserProfile (`$HOME` is not a Windows variable) means it will only apply to the current machine.

That said, most people aren't using roaming accounts, so will likely never notice. The Windows Store package has its own roaming space, accessed in the same way, that _will_ roam for most people (including automatic backups, etc.), but without some kind of Python app for updating the file it'll likely never be set.

Reply:
Great, thanks for the explanation. Committed your change suggestion.